frontend/components/markup-modal/markup-modal-content.js

```diff
@@ -31,7 +31,8 @@ export default class MarkupModalContent extends React.Component {
       example: {
         title,
         example: { pattern, namedParams, queryParams },
-        preview: { style: initialStyle },
+        link,
```

Comment:
I think this `link` property can be removed. As far as I can tell `<Customizer>` doesn't (or doesn't need to) use `props.link` since that value is in `props.exampleQueryParams.link` instead.